source/frontend/src/features/mapSideBar/compensation/list/CompensationListContainer.tsx

```diff
@@ -72,6 +77,15 @@ export const CompensationListContainer: React.FC<ICompensationListContainerProps
 
               updatedFileResponse = await putAcquisitionFile(updatedAcquisitionFile, []);
             }
+            break;
+          case ApiGen_CodeTypes_FileTypes.Lease:
+            {
+              const updatedLease = file as ApiGen_Concepts_Lease;
+              file.totalAllowableCompensation = totalAllowableCompensation;
```

Comment:
I think this works because updatedLease and file point to the same reference, but probably would be clearer to say updatedLease.totalAllowableCompensation

Reply:
updated